telemetry/definitions/commonDefinitions.json

```diff
@@ -4398,6 +4408,34 @@
                 }
             ]
         },
+        {
+            "name": "codewhisperer_codeScanIssueGenerateFixDiffUse",
```

Comment:
can we add `findingId`?

Reply:
there is already `codewhisperer_codeScanIssueApplyFix` . why can't "diff" be a field (name TBD) on that existing metric?

Reply:
Added the codeFixAction to `codewhisperer_codeScanIssueApplyFix`